rustls/src/crypto/aws_lc_rs/mod.rs

```diff
@@ -222,6 +228,12 @@ static SUPPORTED_SIG_ALGS: WebPkiSupportedAlgorithms = WebPkiSupportedAlgorithms
             SignatureScheme::RSA_PKCS1_SHA256,
             &[webpki_algs::RSA_PKCS1_2048_8192_SHA256],
         ),
+        #[cfg(all(feature = "aws-lc-rs-unstable", not(feature = "fips")))]
```

Comment:
I feel like this is -- even behind a feature named "unstable" -- pretty aggressive for the core crate.  WDYT about doing this in the rustls-post-quantum crate (in a new major version?) I realise it is quite a bit more work and wrangling.

Reply:
Makes sense, will redo!